scio-core/src/test/scala/com/spotify/scio/coders/CoderTestUtils.scala

```diff
@@ -30,4 +37,31 @@ object CoderTestUtils {
     val result = CoderUtils.decodeFromByteArray(reader, bytes)
     result == value
   }
+
+  case class ZstdTestCaseClass(a: Int, b: String, c: Long)
+
+  def writeZstdBytes(bytes: Array[Byte]): File = {
+    val tmp = Files.createTempFile("zstd-test", ".bin").toFile
+    tmp.deleteOnExit()
+    val fos = new FileOutputStream(tmp)
+    try {
+      fos.write(bytes)
+    } finally {
+      fos.close()
+    }
+    tmp
+  }
+
+  def zstdOpts(
+    className: String,
+    path: String,
+    packageName: String = "com.spotify.scio.coders.CoderTestUtils$",
```

Comment:
I'd rather have `className` only being the full one and possibly an overload with `clazz: Class[_]`